src/lib/components/parameters.svelte

```diff
@@ -45,6 +45,12 @@
     );
   };
 
+  const engineeringMode = () => {
```

Comment:
This command enters engineering mode. Would it be desirable to have an opposite function to _exit_ engineering mode? I believe engineering mode contains a superset of the information in non-engineering mode so it's probably not necessary. I'm just trying to ensure we're not missing a reason to allow disabling engineering mode.

Reply:
Yes as based on the docs  engineering mode is simply more info. and the setting is back to normal after power cycle.
the base function allows disable, so it its needed it can be added.